README.md

```diff
@@ -252,6 +263,86 @@ addEventListener('DOMContentLoaded', function() {
+    </td>
+  </tr>
+  <tr>
+    <td><code>web-vitals.base.js</code></td>
+    <td>--</td>
+    <td>
+      <p>An ES module bundle of the <code>web-vitals</code> library without any polyfills included.</p>
+      Use this bundle if (and only if) you've also added the <code>polyfill.js</code> script to the <code>&lt;head&gt;</code> of your pages. See <a href="#how-to-use-the-polyfill">how to use the polyfill</a> for more details.
+    </td>
+  </tr>
+  <tr>
+    <td><code>polyfill.js</code></td>
+    <td>--</td>
+    <td>
+      A set of small polyfills that expands browser supports (including back/forward cache restores) and fills in some measurements gaps. See <a href="#how-to-use-the-polyfill">how to use the polyfill</a> for more details.
+    </td>
+  </tr>
+</table>
+
+### Which bundle is right for you?
+
+Most developers will generally want to use the "full" bundle (either the ES module or UMD version, depending on your build system), as it's the easiest to use out of the box and integrate into existing build tools.
+
+However, there are a few good reasons to consider using the "base" version along with the `polyfill.js` script. For example:
+
+- FID can be measured in all browsers.
+- FCP, FID, and LCP will be more accurate in some cases (since the polyfill detects the page's initial `visibilityState` earlier).
+
+Also, the minimal polyfill to support back/forward cache restores that is found in the "full" version is largely the same as the code used in the `polyfill.js` script—just split out. This means that using the polyfill gets you wider browser support and more accurate results with almost no increased code cost (just increased implementation complexity).
+
+### How to use the polyfill
+
+Using the polyfill is a two step process:
+
+**1. Inline the code from `polyfill.js` into the `<head>` of your pages.**
+
+For the polyfill to work, it must be added to the `<head>`. The polyfill adds event listeners and records initial page visibility state, and that must happen before any other code runs or the page is rendered.
+
+The polyfill is quite small (~0.5 KiB, gzipped), so it can be inlined to avoid a blocking request.
+
+**2) Import the "base" build of the library**
+
+In your application code, import the "base" build rather than the "full" build. To do this, change any `import` statements to reference `web-vitals/base` rather than `web-vitals`:
+
+```diff
+- import {getLCP, getFID, getCLS} from 'web-vitals';
++ import {getLCP, getFID, getCLS} from 'web-vitals/base';
+```
+
+All other usage instructions (as well as the public API) are the same in both versions.
+
+_**Note:** while it's certainly possible to copy and paste the code in `polyfill.js` directly into your templates (for step #1 above), it's better to automate this within your build process—otherwise you risk the polyfill and base scripts getting out of sync when new versions are released._
+
```

Comment:
I wonder if this section could/should be moved up to the `Usage` section? For example, something like:

----

### Usage

There are two ways to use the `web-vitals` library on your site.

**1. Use the "full" version**

```js
import {getLCP, getFID, getCLS} from 'web-vitals';

...
```

**2. Use the "base" version with `polyfill.js`**

In your application code, import the "base" build rather than the "full" build. To do this, change any `import` statements to reference `web-vitals/base` rather than `web-vitals`:

```diff
- import {getLCP, getFID, getCLS} from 'web-vitals';
+ import {getLCP, getFID, getCLS} from 'web-vitals/base';
```

Then inline the code from `dist/polyfill.js` into the `<head>` of your pages. 

```html
<!DOCTYPE html>
<html>
  <head>
    <script>
      // Inline code from `dist/polyfill.js` here
    </script>
  </head>
  <body>
    ...
  </body>
</html>
```

See <a href="#which-bundle-is-right-for-you">Which bundle is right for you?</a> for the differences between the two options.

Reply:
Feel free to ignore of course if you would rather not move the section up